src/Type/ObjectType/RootQuery.php

```diff
@@ -683,11 +683,11 @@ public static function register_post_object_fields() {
 								break;
 							case 'source_url':
 								$url     = $args['id'];
-								$post_id = attachment_url_to_postid( $url );
+								$post_id = attachment_url_to_postid( $url ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.attachment_url_to_postid_attachment_url_to_postid
```

Comment:
Another one where we could do a if/else strategy with `wpcom_vip_attachment_url_to_postid`